script/tool/lib/src/publish_command.dart

```diff
@@ -196,6 +201,39 @@ class PublishCommand extends PackageLoopingCommand {
           .toList();
 
       for (final pubspecPath in changedPubspecs) {
+        // Read the ci_config.yaml file if it exists
+
+        final String packageName = p.basename(p.dirname(pubspecPath));
+        bool isBatchReleasePackage;
+        try {
+          final File ciConfigFile = packagesDir.fileSystem
```

Comment:
this may need to wait for https://github.com/flutter/packages/pull/10485 where there is a new api to access batch release flags